model/src/main/kotlin/PackageCuration.kt

```diff
@@ -74,6 +91,8 @@ data class PackageCuration(
             it.showStackTrace()
         }.getOrDefault(false)
 
+    private fun String.isVersionRange() = versionRangeIndicators.any { contains(it, ignoreCase = true) }
```

Comment:
Technically you could use `ignoreCase = false` because none of the indicators are letters.

Reply:
`.x` is.

Reply:
Missed that.